src/AspNet.Security.OAuth.DeviantArt/DeviantArtAuthenticationExtensions.cs

```diff
@@ -6,27 +6,23 @@
 
 using System;
 using AspNet.Security.OAuth.DeviantArt;
-using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
-using Microsoft.Framework.OptionsModel;
 
 namespace Microsoft.AspNet.Builder {
     public static class DeviantArtAuthenticationExtensions {
-        public static IServiceCollection ConfigureDeviantArtAuthentication(
-            [NotNull] this IServiceCollection services,
-            [NotNull] Action<DeviantArtAuthenticationOptions> configuration) {
-            return services.Configure(configuration);
-        }
-
-        public static IApplicationBuilder UseDeviantArtAuthentication([NotNull] this IApplicationBuilder app) {
-            return app.UseMiddleware<DeviantArtAuthenticationMiddleware>();
+        public static IApplicationBuilder UseDeviantArtAuthentication(
```

Comment:
These changes seem unrelated. Make sure to run `git fetch upstream` + `git rebase upstream/dev` + `git push -f` to rebase your branch on top of the latest dev state :+1: